### PR TITLE
Only create TerminalStore after ThemeStore

### PR DESCRIPTION
**Pull request by Nokel81**
- Use Proxy for backwards compatability

Signed-off-by: Sebastian Malton <sebastian@malton.name>

Without this an uncaught exception from mobx happens.

### CODE REVIEW

File: src/renderer/bootstrap.tsx [hunk 1]
```diff
@@ -89,18 +90,22 @@ export async function bootstrap(App: AppComponent) {
...
-  await ClusterStore.createInstance().loadInitialOnRenderer();
+  const cs = ClusterStore.createInstance();
+
+  await cs.loadInitialOnRenderer();
+
   HotbarStore.createInstance();
   ExtensionsStore.createInstance();
   FilesystemProvisionerStore.createInstance();
   ThemeStore.createInstance();
+  TerminalStore.createInstance();
```
Comment (jim-docker) [inline src/renderer/bootstrap.tsx:107]: Maybe a comment indicating that the order is important here?

Comment (Nokel81) [reply src/renderer/bootstrap.tsx:107]: Good idea.

**Comment by Nokel81** [comment]
@jim-docker I have added comments and fixed all unit tests. Testing a `make dev` build it seems to work!

**Comment by jim-docker** [comment]
> * Use Proxy for backwards compatability
> 
> Signed-off-by: Sebastian Malton [sebastian@malton.name](mailto:sebastian@malton.name)
> 
> Without this an uncaught exception from mobx happens.

Is there an issue? Does it always happen?

**Comment by Nokel81** [comment]
The mobx error is a race condition. To reproduce have an open local terminal session with the dock open and then restart lens.

**Comment by Nokel81** [comment]
@jim-docker This is the error I see 
Screen Shot 2021-07-13 at 12 32 56 PM

**Review by jim-docker — APPROVED** [review]
I managed to reproduce and then confirm that this fixes it

**Pull request merged**